packages/react/src/time-picker/TimePicker.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import { useCombobox } from 'downshift';
+import { type ComboboxProps } from '../combobox/Combobox';
+import { ComboboxBase } from '../combobox/ComboboxBase/ComboboxBase';
+import {
+	type DefaultComboboxOption,
+	useComboboxInputId,
+} from '../combobox/utils';
+import { type TimeFormat } from '../time-input/utils';
+import { filterOptions, generateOptions } from './utils';
+
+export function TimePicker({
+	id,
+	inputRef: inputRefProp,
+	interval = 15,
```

Comment:
Setting this to `1` makes the component quite slow. Do we need to virtualise for this sort of scenario? Are we expecting this level of fine-tuned input?

Reply:
We're not expecting anyone needing that much granularity. The preview url is much more performant than dev though. The whole combobox is slow in dev (including the hover states). We should look into that

Reply:
Yeah, performance improvements would be good in the combobox. For a later date.